www/source/docs/habitat-and-other-software.html.md.erb

```diff
@@ -0,0 +1,60 @@
+---
+title: Habitat Docs - Habitat and Other Software
+description: Describes how Habitat works with other popular software technologies.
+---
+# <a name="installing-packages" id="installing-packages" data-magellan-target="installing-packages">Habitat and Other Software</a>
+Habitat helps build, deploy, and manage your applications. To help understand Habitat, we explore how Habitat integrates, complements, or overlaps with other popular software.
+
+### Table of Contents
+
+  - [Habitat and Provisioning Tools](#habitat-and-provisioning)
+  - [Habitat and Containers](#habitat-and-containers)
+  - [Habitat and Container Orchestration](#habitat-and-orchestration)
+  - [Habitat and Continous Integration](#habitat-and-continuous-integration)
```

Comment:
`Continuous`

Reply:
If you look at that word long enough, it starts to look wrong even when it isn't. 😄